index.html

```diff
@@ -183,13 +197,8 @@
         *   If this is a single-line print statement, just execute it.
         */
         // line = `${}`
-      } else if (line.match(/^\w+$/m)) {
```

Comment:
Why did you remove this? This is needed to get single-line statements to display correctly. In your version, this will display like a tuple:

```python
>>> a = [1, 2, 3]
>>> a
(1, 2, 3)
```

Reply:
I removed it because the RegEx would end up matching some of the line text. But, in the recent commit (https://github.com/daniel-schroeder-dev/cwhq-python-repl-hacking/pull/1/commits/8e2898111491428863b120d9641a038ebd19e755) I added it back.